### ISSUE TITLE
Meet Classic Block notification

### ISSUE THREAD

**Issue by arcangelini** [Automattic/wp-calypso#51155]
### Steps to reproduce the behavior

1. Go to WordPress editor
2. View the "Meet the Classic Block" message and click through to dismiss it.
3. Close the page and log out of WordPress.com.
4. Log in to WordPress.com again and go to the page editor
5. View the "Meet the Classic Block" message again

#### What I expected to happen
Once this has been accepted and acknowledged this should no longer be shown. 

#### What actually happened
The message appears every time after logging out and back in. I think this based on a cookie? but if that cookie is purged or removed the message continues to display. This becomes annoying if you edit on multiple devices or are constantly clearing browsing data during testing.

[![Screenshot](https://d.pr/i/Dzk9vQ+)](https://d.pr/i/Dzk9vQ)

Ideally, it would be nice if we could have a way to opt-out of this from the notifications section maybe? In addition to the cookie for cases like this.

### Context
3819867-zen

**Comment by creativecoder** [comment]
Whether this classic block welcome guide is shown is stored in local storage for the specific browser you are using, so it will show again across browsers, loading the editor for a different site, or if caches are cleared.

Regardless, it's probably time to remove this guide since it's been several months since the Calypso editor was deprecated (see https://github.com/Automattic/wp-calypso/issues/46472). The more generic WP.com editor welcome guide has a better mechanism for tracking when a user sees it across sites and browsers, and won't keep reappearing.

**Comment by inaikem** [comment]
@creativecoder, agreed. We also have the Unified Nav modal in place, too, which should be a higher priority to get in front of users than this IMHO.
